### ISSUE TITLE
Total number of aptamers in file: 0 /n Total number of aptamers: 0

### ISSUE THREAD

**Issue by mengops** [drivenbyentropy/aptasuite#89]
[log_2020-05-12_11-31-28.txt](https://github.com/drivenbyentropy/aptasuite/files/4614674/log_2020-05-12_11-31-28.txt)
[log_2020-05-12_11-32-07.txt](https://github.com/drivenbyentropy/aptasuite/files/4614676/log_2020-05-12_11-32-07.txt)

After importing the data using the new experiment window on the GUI version of aptasuite,
I always get empty "Experiment Overview" and "Sequencing Data" windows, the "Aptamer Pool" window is hanging and aptaCluster does not run.

the fastq file is in the following format

@B1_revcomp/1-109 1:
TAGGGAAGAGAAGGACATATGATAGTCTCGCCAAAACACAAATCTCGGAGAAGTTGGAAACAGTTGACTAGTACATGACCACTTGAAAGGGCGAATTCGCGGCCGCTAA
+
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
@B3_revcomp/1-109 1:
TAGGGAAGAGAAGGACATATGATAACATAAGTGTGAGGCTCGATCCTTAGTAATCGTCGTATGTTATCTATGACATGCTCTGTGGTTGACTAGTACATGACCACTTGAA
+
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII

Is there any reason why the aptamer sequences can not be detected by the software?
many thanks in advance for your help

kind Regards
Martina

**Comment by drivenbyentropy** [comment]
Hi,

Could this be an encoding issue of the fastq file? AptaSuite expects UTF-8 as its default encoding. Some preprocessing tools however, appear to change the files encoding when used prior to AptaSuite. See https://github.com/drivenbyentropy/aptasuite/issues/12#issuecomment-366358734 for more details. 

Thanks!

**Comment by mengops** [comment]
Hi
Many thanks for your reply and help and apologies for bothering, but it is my first time working from command line and with anything like this so I am not really capable of pinpointing errors.

First, I followed your suggestion and checked the encoding of the files on notepad++ (on windows) and they both are UTF-8 enconded.

Just to provide further information I am running the precompliled java GUI interface  of Aptasuite v0.9.4. from a Ubuntu virtual machine. I allowed 40GB of memory for running the java file. I am currently adding only the aptamer sequences of one file. Using the same primer sequences for both the general 5' and 3' primers and the Index 5' and 3' (info of each specific cycle)

the first log file also shows the following error
11:31:28 | WARNING | JavaFX Application Thread]: utilities.Version
Error tokenizing JSON response from version request 
could that have anything to do with this? or should I not worry about it?

the same log file also shows
[11:31:42 | CONFIG | pool-2-thread-1]: lib.parser.aptaplex.AptaPlexProducer
Added poison pill to parsing queue 
[11:31:42 | CONFIG | pool-2-thread-3]: lib.parser.aptaplex.AptaPlexConsumer
Encountered poison pill. Exiting thread. 
[11:31:42 | CONFIG | pool-2-thread-1]: lib.parser.aptaplex.AptaPlexConsumer
Encountered poison pill. Exiting thread. 
[11:31:42 | CONFIG | pool-2-thread-2]: lib.parser.aptaplex.AptaPlexConsumer
Encountered poison pill. Exiting thread. 
[11:31:42 | CONFIG | Thread-4]: lib.aptamer.datastructures.MapDBAptamerPool
Closing pool file handles. 

and the parsing completed in 0.546 seconds. 

Following the comments of the previous post i also checked that all files are saved locally and I can confirm that.
I have also uploaded the file in txt format and the configuration file. Hope it helps
[configuration.aptasuite.txt](https://github.com/drivenbyentropy/aptasuite/files/4616175/configuration.aptasuite.txt)


[Sequences.txt](https://github.com/drivenbyentropy/aptasuite/files/4616170/Sequences.txt)
